subsys/testsuite/ztest/include/zephyr/ztest_test.h

```diff
@@ -344,8 +358,17 @@ struct ztest_unit_test *z_ztest_get_next_test(const char *suite, struct ztest_un
 
 /* definitions for use with testing application shared memory   */
 #ifdef CONFIG_USERSPACE
+/**
+ * @brief Make memory used by Ztest userspace accessible
+ */
 #define ZTEST_DMEM K_APP_DMEM(ztest_mem_partition)
+/**
+ * @brief Make memory used by Ztest userspace accessible
+ */
 #define ZTEST_BMEM K_APP_BMEM(ztest_mem_partition)
```

Comment:
would be better to use two actually different briefs to explain differnece between BMEM and DMEM version?

Reply:
fixed